dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresDbClient.cs

```diff
@@ -15,6 +15,11 @@ namespace Microsoft.SemanticKernel.Connectors.Postgres;
 /// <summary>
 /// An implementation of a client for Postgres. This class is used to managing postgres database operations.
 /// </summary>
+/// <remarks>
+/// This class is used with the PostgresMemoryStore, which is being deprecated.
+/// Use the <see cref="PostgresVectorStoreDbClient"/> class with the PostgresVectorStore
```

Comment:
Same comment as above, `PostgresVectorStoreDbClient` is internal. `PostgresVectorStore` and/or `PostgresVectorStoreRecordCollection` can be used as a new reference points.